scripts/tests

```diff
@@ -345,7 +345,13 @@ def kat(force_qemu, verbose, cflags, arch_flags):
     default="NO",
     help="Method for counting clock cycles. PMU requires (user-space) access to the Arm Performance Monitor Unit (PMU). PERF requires a kernel with perf support.",
 )
-def bench(force_qemu, verbose, cycles, cflags, arch_flags):
+@click.option(
+    "-r",
+    "--resultfile",
+    nargs=1,
+    help="Path to result file in json format",
+)
```

Comment:
I'm thinking maybe it will be better to name the flag as `output` or `output-file`, and use `-o` for the short alias ?